examples/helloworld-cnab/bundle.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "helloworld",
+  "version": "0.1.1",
+  "description": "A short description of your bundle",
+  "keywords": [
+    "helloworld",
+    "cnab",
+    "tutorial"
+  ],
+  "maintainers": [
+    {
+      "name": "Jane Doe",
+      "email": "jane.doe@example.com",
+      "url": "https://example.com"
+    }
+  ],
+  "invocationImages": [
+    {
+      "imageType": "docker",
+      "image": "cnab/helloworld:0.1.1"
+    }
+  ],
+  "images": null,
```

Comment:
nit: remove this null keys?

Reply:
I reference them in the doc so it might make sense to leave them. It would also be good to add another example later with an image list.